frontend/public/components/utils/managed-by.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+
+import { compareKinds, ResourceLink } from './resource-link';
+import { K8sResourceKind, referenceForOwnerRef } from '../../module/k8s';
+import { RootState } from '@console/internal/redux';
+import { withFallback } from '@console/shared/src/components/error/error-boundary';
+
+const managerStateToProps = (state: RootState): managerStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
+    receivedCSVs: state.k8s.getIn(['CSVS', 'csvs']),
+  };
+};
+
+const namespacedCSVManager = withFallback<CSVManagerProps>(
+  ({ activeNamespace, className, obj, receivedCSVs = [] }) => {
+    const ownerReferences = _.get(obj.metadata, 'ownerReferences') || [];
+    const namespacedCSVs = receivedCSVs.filter((csv) => csv.metadata.namespace === activeNamespace);
+    const owners =
+      receivedCSVs !== null
+        ? ownerReferences.filter((owner) => namespacedCSVs.some((csv) => compareKinds(csv, owner)))
+        : [];
+
+    if (owners.length === 1) {
+      return (
+        <div className={classNames('co-m-pane__heading-owner', className)}>
+          Managed by{' '}
+          <ResourceLink
+            key={owners[0].uid}
+            kind={referenceForOwnerRef(owners[0])}
+            name={owners[0].name}
+            namespace={activeNamespace}
+            owner={owners[0]}
+          />
+        </div>
+      );
+    }
+    return null;
+  },
+);
+
+export const CSVManager = connect(managerStateToProps)(namespacedCSVManager);
```

Comment:
I'd suggest a name other `CSVManager` (this isn't managing CSVs). Maybe `ManagedByOperatorLink`?